### PR TITLE
[cmd] Add new command line options to sort run data

### PR DESCRIPTION
**Pull request by csordasmarton**

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- web/tests/functional/cmdline/test_cmdline.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=4906598&bid=16245742)

File: docs/web/user_guide.md [hunk 1]
```diff
@@ -783,6 +783,9 @@ optional arguments:
                         'year:month:day:hour:minute:second' (the "time" part
                         can be omitted, in which case midnight (00:00:00) is
                         used).
+  --sort {date,duration,cc_version,name,unresolved_reports}
```
Comment (gyorb) [inline docs/web/user_guide.md]: `codechecker_version` instead of `cc_version` to be more explicit?

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- web/tests/functional/cmdline/test_cmdline.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=4906598&bid=16245742)

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- web/tests/functional/cmdline/test_cmdline.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=4906598&bid=16245742)

**Comment by csordasmarton** [comment]
@gyorb Done.

**Pull request merged**